crescent/internal/registry.py

```diff
@@ -139,7 +139,7 @@ def build_commands(self) -> Sequence[AppCommand]:
             if command.metadata.deprecated:
                 continue
 
-            if command.metadata.sub_group:
+            elif command.metadata.sub_group:
```

Comment:
This is a bad change. No reason to do if > elif if the code will never be reached anyway.